source/draw/Color.ooc

```diff
@@ -14,124 +14,67 @@ ColorMonochrome: cover {
 	normalized ::= this y as Float / 255
 	init: func@ (=y)
 	init: func@ ~default { this init(0) }
-	init: func@ ~uint (i: UInt) { this init(i as Byte) }
-	init: func@ ~uint16 (i: UShort) { this init(i as Byte) }
-	init: func@ ~uint32 (i: UInt) { this init(i as Byte) }
-	init: func@ ~uint64 (i: ULong) { this init(i as Byte) }
-	init: func@ ~int (i: Int) { this init(i as Byte) }
-	init: func@ ~float (f: Float) { this init(f*255.0f clamp(0.0f, 255.0f) as Byte) }
-	init: func@ ~double (d: Double) { this init(d*255.0f clamp(0.0f, 255.0f) as Byte) }
-	set: func@ (color: This) {
-		this y = color y
-	}
-	copy: func -> This { this }
-	toMonochrome: func -> This { this copy() }
+	set: func@ (color: This) { this y = color y }
```

Comment:
What is the intended use case?
Seems like I might as well use `color y = 255` or `color = ColorMonochrome new(255)`.